mne/viz/epochs.py

```diff
@@ -825,11 +835,18 @@ def plot_epochs_psd(epochs, fmin=0, fmax=np.inf, proj=False, n_fft=256,
 
 def _plot_traces(params):
     """ Helper for plotting concatenated epochs """
+    params['text'].set_visible(False)
     ax = params['ax']
-    n_channels = params['n_channels']
+    butterfly = params['butterfly']
+    if butterfly:
```

Comment:
That's actually against what PEP8 says. Is there a particular reason why 'is True' should be added?